.github/workflows/ci.yml

```
@@ -3,6 +3,7 @@ name: 🔍 코드 품질 검사
 on:
   push:
     branches: [ main, develop ]
+    branches: [ main, develop ]
```

actionlint (1.7.7)

key "branches" is duplicated in "push" section. previously defined at line:5,col:5

(syntax-check)